.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,74 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees:
+  - octocat
```

Comment:
should this be one of us?

Reply:
Agreed with this assessment, looks like an array, just put all our usernames